src/comments/ignore.js

```diff
@@ -1,16 +1,31 @@
-function ignoreComments(node) {
-  if (node === undefined || node === null) return;
+function ignoreComments(path) {
+  const node = path.getValue();
+  // We ignore anything that is not an object
+  if (node === null || typeof node !== 'object') return;
```

Comment:
do we wanna keep `node === undefined ||` as well?

Reply:
It's not necessary because of the object type check.
```Javascript
typeof undefined !== 'object' // true

typeof null !== 'object' // false
```